### PR TITLE
Adding sample NodeJS scenario test

### PR DESCRIPTION
**Pull request by hukoyu**
This is a sample hello world NodeJS test scenario.

This PR can be merged after 
this PR https://github.com/lsds/sgx-lkl/pull/115 merged which addresses bug https://github.com/lsds/sgx-lkl/issues/103 that blocks running any nodejs app.

cc @letmaik @prp @davidchisnall

### CODE REVIEW

**Comment by hukoyu** [comment]
@letmaik blocking PR https://github.com/lsds/sgx-lkl/pull/115 is merged. I resynced to upstream and NodeJS scenario test runs and completes successfully. This is the first NodeJS test and ready to be merged.

File: tests/languages/nodejs/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+PROG=/app/index.js
+ENCLAVE_CMD=usr/local/bin/node ${PROG}
+
+DISK_IMAGE=sgxlkl-nodejs.img
+IMAGE_SIZE=1.2G
+
+ifeq ($(SGXLKL_VERBOSE),)
+SGXLKL_ENV=\
+   SGXLKL_MMAP_FILES=Shared SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=0 \
```
Comment (letmaik) [inline tests/languages/nodejs/Makefile:11]: Is `SGXLKL_MMAP_FILES=Shared` required to run nodejs? If so, why is it not set in the else branch?

File: tests/languages/nodejs/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+include ../../common.mk
+
+PROG=/app/index.js
+ENCLAVE_CMD=usr/local/bin/node ${PROG}
```
Comment (letmaik) [inline tests/languages/nodejs/Makefile]: Leading `/` missing?

**Review by letmaik — CHANGES_REQUESTED** [review]
Looks good, just minor comments.

File: tests/languages/nodejs/Makefile (hunk 1 above)
Comment (hukoyu) [reply tests/languages/nodejs/Makefile:11]: Added to else block as well.

File: tests/languages/nodejs/Makefile (hunk 2 above)
Comment (hukoyu) [reply tests/languages/nodejs/Makefile]: It was working since app runs in root folder. But added, it is better like /usr/...

**Comment by hukoyu** [comment]
@letmaik addressed your feedback.

**Review by letmaik — APPROVED** [review]

**Pull request merged**